README.md

```diff
@@ -1,2 +1,45 @@
-# kubernetes-instrumentor
-language detector and auto instrumentation micro service for kubernetes.  Inspired by odigos.io
+# Logz.io kubernetes instrumentor
```

Comment:
Please have @nico-shishkin review and put his magic touch on this doc, since it's not easy on the eyes at the moment 😅